cluster/images/hyperkube/master.json

```diff
@@ -12,8 +12,18 @@
               "/hyperkube",
               "controller-manager",
               "--master=127.0.0.1:8080",
+              "--terminated-pod-gc-threshold=100",
```

Comment:
We set it to 100 in cluster/gce/config-test.sh (test clusters) for a specific e2e tests. I suggest leaving it out. It defaults to 12500